frontend/src/main/pages/Recommendations/RecommendationsIndexPage.js

```diff
@@ -1,13 +1,28 @@
 import React from 'react'
+import { useBackend } from 'main/utils/useBackend'; // use prefix indicates a React Hook
 
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import RecommendationsTable from 'main/components/Recommendations/RecommendationsTable';
+import { useCurrentUser } from 'main/utils/currentUser' // use prefix indicates a React Hook
 
 export default function RecommendationsIndexPage() {
 
-    return (
+  const currentUser = useCurrentUser();
+
+  const { data: recommendations, error: _error, status: _status } =
+    useBackend(
+      // Stryker disable next-line all : don't test internal caching of React Query
+      ["/api/recommendations/all"],
+            // Stryker disable next-line StringLiteral,ObjectLiteral : since "GET" is default, "" is an equivalent mutation
+            { method: "GET", url: "/api/recommendations/all" },
```

Comment:
You should change this URL. When I test your branch, I get some errors in API requests. According to your Recommendation controller java file, the endpoint is defined as: `/api/Recommendation/all`
```suggestion
            { method: "GET", url: "/api/Recommendation/all" },
```